wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -89,12 +89,20 @@ private static void setMpiMode(Properties lmiProperties) {
     }
 
     private static void setTensorParallelDegree(Properties lmiProperties) {
-        if (lmiProperties.containsKey("option.tensor_parallel_degree")) {
-            return;
-        }
-        String tpDegree = Utils.getenv("TENSOR_PARALLEL_DEGREE", "max");
+        String tpDegree =
+                lmiProperties.getProperty(
+                        "option.pipeline_parallel_degree",
```

Comment:
why are we getting pipeline parallel degree here?

Reply:
Sorry for the typo.